### PR TITLE
Update the project to be able to use Node 10+ instead of Node 10.x

### PR DESCRIPTION
**Pull request by matyasf**
Tested with Node 10 and Node 15. `"type": "module"` throws an error when building with Node 12+

### CODE REVIEW

File: packages/ui-docs-plugin/package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@instructure/config-loader": "^7.3.0",
     "@instructure/ui-docs-client": "^7.3.0",
-    "favicons-webpack-plugin": "^1.0.2",
+    "favicons-webpack-plugin": "^4.2.0",
```
Comment (matyasf) [inline packages/ui-docs-plugin/package.json:28]: This dependency needed to be updated, it had a bug in Node 12+

**Comment by Brailor** [comment]
Doesn't it make sense to merge this into master as well?

**Comment by matyasf** [comment]
> Doesn't it make sense to merge this into master as well?

good idea, but I'd first let us test it on the `V8` branch for a bit, there might be some hidden issues.

**Review by HerrTopi — APPROVED** [review]

**Comment by matyasf** [comment]
i cherry picked this to the `next` branch

**Pull request closed without merging**